vidarr-server/src/main/resources/db/migration/V0010__big_serial_id.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE workflow_run ALTER COLUMN id TYPE bigint;
+ALTER TABLE active_workflow_run ALTER COLUMN id TYPE bigint;
+ALTER TABLE active_operation ALTER COLUMN workflow_run_id TYPE bigint;
```

Comment:
@mlaszloffy can you confirm if active_operation.id should also be switched over to bigint? It's currently at least 212,662,776

Reply:
Good catch, I am seeing the last sequence value at 2,079,044 - but this id will grow quickly too. Makes sense to change this ID too.

Reply:
Will do